### ISSUE TITLE
RDM AF HAT

### ISSUE THREAD

**Issue by marduknunya** [EdenServer/community#608]
Hello my Taru named Nunya completed the rdm af hat quest yesterday.  I went to garlaige, popped the nm and got the nail puller.  I traded the nail puller to the oaken box and got the CS with the elvaan rdm.  I then zoned into north sandoria to finish the quest and did not get a cutscene or my hat.  Enclosed is a screenshot of my inv with the nail puller.   When i defeated miser murphy in fei'yin I obtained the antique coin, however when i traded the coin to the fountain to get the first CS the coin then disapeared from my inv after the CS.  This did not happen when i fought the doll in garlaige last night.  

In addition one of my LS mates advised he had the same issue but back when the server was still kupo.  Several of my other LS mates stated they did the quest recently and had no issues.  

Thank you for your time.

**Comment by marduknunya** [comment]
![nunya_2018 12 28_010411](https://user-images.githubusercontent.com/46229571/50527828-c5c82800-0ab8-11e9-9e48-9c7abd1d85d9.png)
![nunya_2018 12 28_004936](https://user-images.githubusercontent.com/46229571/50527829-c5c82800-0ab8-11e9-88db-9b2c4e73f8bb.png)

**Comment by marduknunya** [comment]
![nunya_2018 12 28_164102](https://user-images.githubusercontent.com/46229571/50528959-7f29fc00-0abf-11e9-97b4-0f9ae0766439.png)

**Comment by marduknunya** [comment]
![nunya_2018 12 28_165642](https://user-images.githubusercontent.com/46229571/50529253-a255ab00-0ac1-11e9-955a-82db336e9c3b.png)

**Comment by Lonewulf724** [comment]
I am in the same boat and I have done everything up to getting the final cs in Northern SanD'oria. The final CS is not triggering for me and thus can not receive the hat. Char - Lonewulf

**Comment by marduknunya** [comment]
Hey can we fix this now that more than one person is having the same issue.  I re did the CS and got the item.  I basically stopped playing because of this and i didnt want to bother the devs again. 
    On Sunday, December 30, 2018, 3:25:00 AM EST, tectal831 <notifications@github.com> wrote:  
 
 
Please double check all stages of quest, talk to each npc again etc, and let us know if that doesnt fix it. This has not been known report, and plenty of people have finished this quest. It is most likely something on user end, but let us know if that doesnt solve the problem

**Comment by godmode** [comment]
This is most likely a very large issue to fix and we will fix it. But it will take time. We believe it’s event related which is not just one file we can change a few settings on and everything is fixed. This is also why it doesn’t always break. So we ask that while we don’t have the event handler fixed yet that people try again and if they are stuck to call a GM to advance them.

**Comment by marduknunya** [comment]
So to be clear, you are telling me to contact a GM and they will grant me the RDM AF hat in my inventory?  I dont think I can get past maat without this hat and therefore stopped playing because I wanted my taru to be able to obtain 75 
    On Tuesday, February 19, 2019, 5:43:05 AM EST, Godmode <notifications@github.com> wrote:  
 
 
This is most likely a very large issue to fix and we will fix it. But it will take time. We believe it’s event related which is not just one file we can change a few settings on and everything is fixed. This is also why it doesn’t always break. So we ask that while we don’t have the event handler fixed yet that people try again and if they are stuck to call a GM to advance them.

**Comment by godmode** [comment]
No, I’m asking you to place a GM ticket if you have tried to progress again and you are stuck so we can progress you in the quest. It doesn’t mean I will put the hat into your inventory but I will look into where you are at and where you should be.

**Comment by marduknunya** [comment]
Hello, I recently logged in to the game and tried to get the af hat.  I went through the steps in southern sandy and have some screen shots.  when i went over to the oaken box today and clicked it (on rdm) the box said there is nothing special here.  I also tried zoning into n.sandy as well and did not get the hat.  

I tried placing a GM call today but i assume the queue is very long.  Thank you for your attention to this matter.
